packages/now-cli/src/util/dev/server.ts

```diff
@@ -533,7 +533,7 @@ export default class DevServer {
     // The default empty `vercel.json` is used to serve all files as static
     // when no `vercel.json` is present
     let configPath = 'vercel.json';
-    let config: NowConfig = this.cachedNowConfig || {
+    let config: NowConfig = (canUseCache && this.cachedNowConfig) || {
```

Comment:
```suggestion
    let config: NowConfig = canUseCache ? this.cachedNowConfig : {
```

Ok I think I understand now. Does this make more sense?

Reply:
Your other suggestion is better I think. See ed0b88b.